### PR TITLE
[risk=no] Delint all source files

### PR DESCRIPTION
**Pull request by rushtong**
Ran eslint on the source tree. Using this PR to test the new dev deploy process too.

### CODE REVIEW

File: src/pages/AdminConsole.js [hunk 1]
```diff
@@ -136,11 +136,21 @@ class AdminConsole extends Component {
...
+}); break;
+      case 'addDataset': this.setState(prev => {
+ prev.showAddDatasetModal = false; return prev;
+}); break;
+      case 'addOntologies': this.setState(prev => {
+ prev.showAddOntologiesModal = false; return prev;
+}); break;
+      case 'electionTimeout': this.setState(prev => {
+ prev.showElectionTimeoutModal = false; return prev;
+}); break;
```
Comment (rushtong) [inline src/pages/AdminConsole.js]: Don't think I like was eslint does here. Will see if I can fix that.

Comment (rushtong) [reply src/pages/AdminConsole.js]: Ok, this is `brace-style`. Will disable that and revert some of these changes.

**Review by danxmoran — APPROVED** [review]
Seems straightforward

**Review by RoriCremer — APPROVED** [review]

**Comment by rushtong** [comment]
Thank you @RoriCremer and @danxmoran!

**Pull request merged**